website/docs/cli/commands/test/index.mdx

```diff
@@ -0,0 +1,368 @@
+  directory and also the current directory when you run `tofu test`. The path should be relative to the current
+  working directory.
+* `-filter=testfile` Specify an individual test file to run. Use this option multiple times to specify more than one
+  file. The path should be relative to the current working directory.
+* `-var 'foo=bar'` Set an input variable of the root module. Specify this option multiple times to add more
+  than one variable.
+* `-var-file=filename` Set multiple variables from the specified file. In addition to this file, OpenTofu automatically
+  loads `terraform.tfvars` and `*.auto.tfvars`. Use this option multiple times to specify more than one file.
+* `-json` Change the output format to JSON.
+* `-no-color` Disable colorized output in the command output.
+* `-verbose` Print the plan or state for each test run block as it executes.
+
+## Directory structure
+
+The `tofu test` command supports two directory layouts:
+
+<Tabs>
+    <TabItem value="flat" label="Flat layout" default>
+        This layout places the <code>*.tftest.hcl</code> test files directly next to the <code>*.tf</code> files they
+        test. There are no rules that each <code>*.tf</code> file must have its own test file, but it is a good practice to follow.
+        <CodeBlock>{FlatLayout}</CodeBlock>
+    </TabItem>
+    <TabItem value="nested" label="Nested layout">
+        This layout places the <code>*.tftest.hcl</code> files in a separate <code>tests</code> directory. Similar to
+        the flat layout, there are no rules that each <code>*.tf</code> file must have its own test file, but it is a
+        good practice to follow.
+        <CodeBlock>{NestedLayout}</CodeBlock>
+    </TabItem>
+</Tabs>
+
+### Testing modules
+
+When testing modules, use one of the above directory structures for each module:
+
+<Tabs>
+    <TabItem value="flat" label="Flat layout" default>
+        With this layout, run <code>tofu test -test-directory=./path/to/module</code> to test the module in question.
+        <CodeBlock>{FlatLayoutModule}</CodeBlock>
+    </TabItem>
+    <TabItem value="nested" label="Nested layout">
+        With this layout, <strong>change your working directory to your module path</strong> and
+        run <code>tofu test</code> to test the module in question.
+        <CodeBlock>{NestedLayoutModule}</CodeBlock>
+    </TabItem>
+</Tabs>
+
+:::tip Hint
+
+You can use the `-filter=sometest.tftest.hcl` option to run a limited set of test files. Use the option multiple
+times to run more than one test file.
+
+:::
+
+## The `*.tftest.hcl` file structure
+
+The testing language of OpenTofu is similar to the main OpenTofu language and uses the same block structure. A test file
+consists of:
+
+* The **[`run` blocks](#the-run-block)**: define your tests.
+* The **[`variables` block](#the-variables-and-runvariables-blocks)** (optional): define variables for all tests in the current file.
```

Comment:
Not 100% certain on this one, but if we only allow a singular `variables` block then this could be :

```suggestion
* A **[`variables` block](#the-variables-and-runvariables-blocks)** (optional): define variables for all tests in the current file.
```